main.go

```diff
@@ -20,6 +19,15 @@ Options:
   -a           <left>, <right>, <center> justification (default: left)
 `
 
+var hFlag *bool
```

Comment:
Completely your call but with flag variables, I typically make them explicit so whenever they're used down in the program, it's understood.  Mainly since they're declared "so far away".